models/comments.models.js

```diff
@@ -10,5 +10,18 @@ exports.selectCommentsByArticle = (articleId) => {
     });
 };
 
+exports.addsNewCommentToArticleId = (articleId, body) => {
+    return db
+        .query(`INSERT INTO comments 
+        (body, article_id, author, votes, created_at) 
+        VALUES ($1, $2, $3, $4, CURRENT_TIMESTAMP) 
```

Comment:
I think if you look in the seed file, the created at will have a default value so no need to worry about inserting it here as it will be done automatically